node/src/BaseClient.ts

```diff
@@ -2981,6 +2982,41 @@ export class BaseClient {
         return this.createWritePromise(createZRemRangeByRank(key, start, end));
     }
 
+    /**
+     * Removes all elements in the sorted set stored at `key` with lexicographical order between `minLex` and `maxLex`.
+     * See https://valkey.io/commands/zremrangebylex/ for more details.
```

Comment:
```suggestion
     * 
     * See https://valkey.io/commands/zremrangebylex/ for more details.
```

Reply:
This seems wholly unnecessary.  We don't add this extra line in a bunch of other docs in this same file.